### DISCUSSION TITLE
Querying user profile

### DISCUSSION THREAD

**Discussion by dgiglio** [tidal-music/discussions#25] [Ideas & Feature requests]
Hi,
the APIs published so far are lacking the capability to explore the subscriber own profile (followed artists , liked albums or tracks, playlists...). Will they be available in future?

Thanks in advance.

**Comment by aliaksei-taliuk** [comment]
Hello @dgiglio! Thanks for asking and welcome to the forum! 

Yes, they will. We're working on it. 

It kind of duplicates this one: https://github.com/orgs/tidal-music/discussions/15#discussioncomment-7227348

Stay tuned.
